### PR TITLE
Prod dockerfile

### PR DESCRIPTION
**Pull request by alexdavid**
Finishes the dockerfile refactor we talked about

### CODE REVIEW

File: src/terraform/index.go [hunk 1]
```diff
@@ -77,13 +76,9 @@ func generateServiceModules(deployConfig types.DeployConfig, serviceProtectionLe
...
 
 func generateServiceModule(serviceName string, awsConfig types.AwsConfig, serviceConfig types.ServiceConfig, imagesMap map[string]string, filename string) (string, error) {
-	command, err := json.Marshal(strings.Split(serviceConfig.Startup["command"], " "))
-	if err != nil {
-		return "", errors.Wrap(err, "Failed to marshal service startup command")
-	}
 	varsMap := map[string]string{
 		"serviceRole":         serviceName,
-		"startupCommand":      string(command),
+		"startupCommand":      "[]",
```
Comment (charlierudolph) [inline src/terraform/index.go]: Could we remove this variable?

**Review by charlierudolph — COMMENTED** [review]
What about the rename to `Dockerfile.prod`?

File: src/terraform/index.go (hunk 1 above)
Comment (alexdavid) [reply src/terraform/index.go]: I removed it from the templated terraform file which should allow it to be defaulted in the terraform modules

**Review by hugoesthere — APPROVED** [review]
Good once tests are green

**Pull request merged**